recipes/libbip3x/meta.yaml

```
@@ -0,0 +1,466 @@
+{% set name = "libbip3x" %}
+{% set version = "3.0.0" %}
+
+{% set so_major_version = version.split('.')[0] %}
+
+{% set bip3x_h = ["bip3x_config.h", "bip3x_crypto.h", "bip3x_hdkey_encoder.h", "bip3x_mnemonic.h"] %}
+{% set cbip3x_h = ["cbip3x_config.h", "cbip3x.h", "cbip3x_hdkey_encoder.h"] %}
+{% set details_h = ["PCGRand.hpp", "uint256.h", "uint256_t.hpp", "utils.h"] %}
+{% set crypto_h = [
+  "address.h", "base58.h", "bignum.h", "bip39_english.h", "bip39.h",
+  "blake256.h", "check_mem.h", "common.h", "ecdsa.h", "hasher.h",
+  "hmac.h", "hmac_sha256.h", "hmac_sha512.h", "memzero.h", "options.h",
+  "packed_attr.h", "pbkdf2.hpp", "rand.h", "rfc6979.h", "ripemd160.h",
+  "secp256k1.h", "secp256k1.table", "sha256.h", "sha2.hpp", "sha3.h",
+  "sha512.h", "win_endian.h",
+] %}
+{% set bip3x_header_paths = {
+  'bip3x': bip3x_h,
+  'bip3x/crypto': crypto_h,
+  'bip3x/details': details_h
+} %}
+package:
+  name: libbip3x-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/edwardstock/bip3x/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ef663ad90c6251452509943e621c337d88ea9afb9e14432dcad1f4bb7b175fb8
+  patches:
+    - patches/0001-add-so-version.patch
+    - patches/0002-add-so-version-c.patch
+    - patches/0003-add-so-version-jni.patch
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja
+    - openjdk
+    - openssl
```

These should be split.
```suggestion
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - {{ stdlib('c') }}
    - cmake
    - ninja
  host:
    - openjdk
    - openssl
```